Add Missing Entries to Available CLI Configurations (#1744)
astronomer/docs: astro/cli/configure-cli.md

| ------------------- | ----------- | --------------- |
| `cloud.api.protocol`  | The type of protocol to use when calling the Airflow API in a local Airflow environment         | `http`, `https`             |
| `cloud.api.port`      | The port to use when calling the Airflow API in a local environment           | Any available port             |

> | `cloud.api.ws_protocol`   | The type of WebSocket (ws) protocol to use when calling the Airflow API in a local Airflow environment           | `http`, `https`             |

Reviewer: default is `wss`

> | `cloud.api.ws_protocol`   | The type of WebSocket (ws) protocol to use when calling the Airflow API in a local Airflow environment           | `http`, `https`             |

Reviewer: ```suggestion
| `cloud.api.ws_protocol`   | The type of WebSocket (ws) protocol to use when calling the Airflow API in a local Airflow environment           | `ws, wss`             |
```

Author: The edit has been added.

> | `cloud.api.token`    | The type of API token to use when calling the Airflow API in a local Airflow environment           | Any available API token             |

Reviewer: Done need this one

> | `cloud.api.token`    | The type of API token to use when calling the Airflow API in a local Airflow environment           | Any available API token             |

Reviewer: ```suggestion
```

Author: The entry has been removed.

| `context`            | The context for your Astro project          | Any available [context](cli/astro-context-list.md)             |

> | `contexts`           | The contexts for your Astro project          | Any available [context](cli/astro-context-list.md)             |

Reviewer: We dont use this one as well, good to remove

> | `contexts`           | The contexts for your Astro project          | Any available [context](cli/astro-context-list.md)             |

Reviewer: ```suggestion
```

Author: The entry has been removed.

| `local.registry`     | The location of your local Docker container running Airflow             | Any available port             |
| `postgres.user`      | Your username for the Postgres metadata database            | Any string             |
| `postgres.password`  | Your password for the Postgres metadata database            | Any string             |
| `postgres.host`      | Your hostname for the Postgres metadata database            | Any string             |
| `postgres.port`      | Your port for the Postgres metadata database            | Any available port             |
| `project.name`       | The name of your Astro project         | Any string             |

> | `project.workspace`       | The name of your Astro project workspace         | Any string             |

Reviewer: We can remove this, since we dont use his config in the cli yet.

Author: Sorry, @kushalmalani I'm having difficulty determining what specific entries should be removed. Can you list them please?

Reviewer: project.workspace, contexts, cloud.api.token can be removed

> | `project.workspace`       | The name of your Astro project workspace         | Any string             |

Reviewer: ```suggestion
```

Author: The entry has been removed.

> | `webserver.port`     | The port for the web server in your local Airflow environment          | Any available port             |

Reviewer: default port is 8080

| `show_warnings`      | Determines whether warning messages appear when starting a local Airflow environment         | `true`, `false`             |

> | `skip_parse`  | Determines whether parsing is ignored when executing a Houston API call         | `true`, `false`             |

Reviewer: Update description to `Setting this config to true will skip parsing of dags when you deploy code to a deployment`

Author: The description was added.

> | `skip_parse`  | Determines whether parsing is ignored when executing a Houston API call         | `true`, `false`             |

Reviewer: ```suggestion
| `skip_parse`  | Determines whether parsing of dags need to be skipped when you deploy code to a deployment        | `true`, `false`             |
```

Author: The edit has been added.

| `cloud.api.protocol`  | The type of protocol to use when calling the Airflow API in a local Airflow environment         | `http`, `https`             |
| `cloud.api.port`      | The port to use when calling the Airflow API in a local environment           | Any available port             |
| `cloud.api.ws_protocol`   | The type of WebSocket (ws) protocol to use when calling the Airflow API in a local Airflow environment           | `http`, `https`             |
| `cloud.api.token`    | The type of API token to use when calling the Airflow API in a local Airflow environment           | Any available API token             |
| `context`            | The context for your Astro project          | Any available [context](cli/astro-context-list.md)             |
| `contexts`           | The contexts for your Astro project          | Any available [context](cli/astro-context-list.md)             |
| `local.astrohub`     | The location of your local Astro project          | `http://localhost:8871/v1`             |
| `local.core`         | The location of your local Airflow scheduler          | `http://localhost:8888/v1alpha1`             |

> | `local.public_astrohub`  | The location of your local, public Astro project          | `http://localhost:8871/graphql`             |

Reviewer: We should not display the following configs since these are internal services and a external user would not have access to this

local.astrohub
local.core
local.public_astrohub

Author: @kushalmalani I removed the entries from the table.

| `local.registry`     | The location of your local Docker container running Airflow             | Any available port             |
| `local.houston`      | The location of your local Houston API instance             | Any string             |

> | `local.platform`      | The location of your local platform             | Any string             |

Reviewer: @paolaperaza Should we add software configs? since this page is dedicated to astro-cli with Astro

Reviewer 2: Yea, let's pull those out of this list. @kushalmalani Other than the obvious Houston ones, can you send a list here of the Software-only configs?

Reviewer: We can remove local.houston, local.platform, houston.dial_timeout, houston.skip_verify_tls

@kwcanuck

Author: @kushalmalani I removed the entries as recommended. Please confirm the accuracy of the definitions and values.
